redwood-treehouse-lazylayout-composeui/src/commonMain/kotlin/app/cash/redwood/treehouse/lazylayout/composeui/ComposeUiLazyColumn.kt

```diff
@@ -54,9 +55,20 @@ internal class ComposeUiLazyColumn<A : AppService>(
     ) {
       intervals.forEach { interval ->
         items(interval.count) { index ->
-          Box(Modifier.height(64.dp)) {
-            TreehouseContent(treehouseApp, widgetSystem) { interval.itemProvider.get(index) }
+          var isPlaceholderVisible by remember { mutableStateOf(true) }
+          if (isPlaceholderVisible) {
+            placeholder.render()
+          }
+          val codeListener by remember {
+            mutableStateOf(
+              object : TreehouseView.CodeListener() {
+                override fun onCodeLoaded(initial: Boolean) {
+                  isPlaceholderVisible = false
+                }
+              },
+            )
           }
```

Comment:
I think mutablestate is unnecessary 
```suggestion
          val codeListener = remember {
           
              object : TreehouseView.CodeListener() {
                override fun onCodeLoaded(initial: Boolean) {
                  isPlaceholderVisible = false
                }
              },
            
           }
```

Reply:
Good catch!